packages/perseus/src/index.ts

```diff
@@ -151,7 +151,7 @@ export type {ParsedValue} from "./util";
 export type {
     LockedFigure,
     LockedFigureType,
-    LockedPoint,
+    LockedPointType,
```

Comment:
I added that suffix because in the refactor, I created a `LockedPoint` component for the actual locked point on the mafs graph. Typescript didn't like them having the same name.